src/benchcab/benchcab.py

```diff
@@ -259,9 +259,7 @@ def build(self, config_path: str, mpi=False):
                 self.logger.info(
                     f"Compiling CABLE {build_mode} for realisation {repo.name}..."
                 )
```

Comment:
Isn't this comment wrong now? It seems we compile either serially or serially and with mpi. Should we have:
```
                build_mode = "with MPI and " if mpi else ""
                self.logger.info(
                    f"Compiling CABLE {build_mode} serially for realisation {repo.name}..."
                )
```

Reply:
Done